docs/content/preview/develop/common-patterns/timeseries/ordering-by-entity.md

```diff
@@ -51,24 +51,26 @@ INSERT INTO entity_order1 (ts, car, speed)
 Retrieve some data from the table as follows:
 
 ```sql
-SELECT *, yb_hash_code(car) % 3 as tablet FROM entity_order1;
+SELECT * FROM entity_order1 WHERE car = 'car-1' ;
 ```
 
 ```output
-        ts          |  car  | speed | tablet
----------------------+-------+-------+--------
- 2023-07-01 00:00:03 | car-2 |    50 |      1
- 2023-07-01 00:00:05 | car-2 |     8 |      1
- 2023-07-01 00:00:07 | car-2 |    19 |      1
- 2023-07-01 00:00:09 | car-2 |    56 |      1
- ...
- 2023-07-01 00:00:01 | car-1 |     2 |      2
- 2023-07-01 00:00:02 | car-1 |    38 |      2
- 2023-07-01 00:00:04 | car-1 |    51 |      2
- 2023-07-01 00:00:06 | car-1 |    10 |      2
+         ts          |  car  | speed
+---------------------+-------+-------
+ 2023-07-01 00:00:01 | car-1 |    22
+ 2023-07-01 00:00:03 | car-1 |    43
+ 2023-07-01 00:00:04 | car-1 |    11
+ 2023-07-01 00:00:07 | car-1 |    39
+ 2023-07-01 00:00:08 | car-1 |    21
+ 2023-07-01 00:00:14 | car-1 |     5
+ 2023-07-01 00:00:15 | car-1 |    31
+ 2023-07-01 00:00:16 | car-1 |    21
+ 2023-07-01 00:00:18 | car-1 |    14
+ 2023-07-01 00:00:19 | car-1 |    46
+ 2023-07-01 00:00:20 | car-1 |     4
 ```
 
-Notice that the data for each car is together (in the same tablet), but at the same time, the data is automatically sorted. The key thing to note here is that all the data for a specific car (say `car-1`) will be located in the same tablet (`2`), because you have defined the data to be distributed on the hash of `car` (`PRIMARY KEY(car HASH, ts ASC)`).
+Notice that the data for `car-1` is together, but at the same time, the data is automatically sorted. The key thing to note here is that all the data for a specific car (say `car-1`) will be located in the same tablet (`2`), because you have defined the data to be distributed on the hash of `car` (`PRIMARY KEY(car HASH, ts ASC)`).
```

Comment:
"the data for `car-1` is together" - but you only asked for car-1, so in what sense is it together?
automatically sorted - by time I guess you mean?
"will be located in the same tablet" - how do I know?  you are just telling, not showing, I guess

Reply:
Fixing the text